piton/design/chip/tile/l15/rtl/l15_pipeline.v.pyv

```diff
@@ -640,6 +640,47 @@ begin
                         predecode_address_s1 = 0; // bug fix, address cannot be X's
                     end
 
+                end
+                `MSG_TYPE_NC_LOAD_MEM_ACK:
```

Comment:
Wait does this just duplicate `MSG_TYPE_DATA_ACK`? If so, you can save the duplication by extending the `MSG_TYPE_DATA_ACK` case above like so:

```
-`MSG_TYPE_DATA_ACK:
+`MSG_TYPE_DATA_ACK, `MSG_TYPE_NC_LOAD_MEM_ACK:
```

Reply:
I was thinking there might be some difference in the beginning